### PR TITLE
Issue-22: Fix issue with note text not getting exported in report.  B…

### PR DESCRIPTION
**Pull request by wonderificdave**
…oolean from the ternary expression was getting exported instead

### CODE REVIEW

**Comment by asalant** [comment]
This one doesn't make sense to me. The change you made looks like the identical logic to the ternary expression you replaced - "If the note is nil, append nil. Otherwise append note.text."

What was the issue you were trying to fix? Can you reproduce it with a test that fails with the old code and passes with the new?

**Comment by wonderificdave** [comment]
Hi Alon, can you take a look at this one again?  I added a test and also simplified the fix... it is an order of operations problem.  The new fix is similar to the code in visit.rb line 64

**Comment by wonderificdave** [comment]
Hi Alon,

Can you take a look at this one again?  I put a fix in a few weeks ago and
wanted to make sure you saw it.  Let me know if you have any questions
-dave

**Comment by asalant** [comment]
Thanks for the poke. I started to check it out on a new laptop and got hung
up on @$!#@! Rails/Yosemite setup. I'll look again this week. Don't
hesitate to poke me again.

File: app/models/service.rb [hunk 1]
```diff
@@ -74,7 +74,7 @@ def to_csv
     values << end_date ? nil : end_date.to_s(:db)
     values << volunteered
     values << paid
-    values << note.nil? ? nil : note.text
+    values << (note.nil? ? nil : note.text)
```
Comment (asalant) [inline app/models/service.rb]: Hah! Great fix. Esp with the test.

**Comment by asalant** [comment]
Resolved in PR #28.

**Comment by wonderificdave** [comment]
Hi Alon,

Looks like the freehub site is down with a 500 error.  Have you seen this?
Let me know if there's anything I can do to help

Thanks!
-dave

**Comment by asalant** [comment]
Hey Dave, thanks for the heads up and the offer to help. I'm on it but
would be great to be get in touch directly. Please send me your email
address to alon at salant.org. Thanks!

**Pull request closed without merging**